### PR TITLE
Revise user-level documentation workflow

### PR DESCRIPTION
**Pull request by sarakonradi**
This PR revises the user-level documentation workflow. It:

- Renames "user" to "user-level"
- Mentions the concept of "user-correctable documentation"
- Uses the active voice
- Breaks the lines

### CODE REVIEW

File: doc/documentation_workflow/user_documentation_workflow.rst [hunk 1]
```diff
@@ -114,17 +138,24 @@ Proceed as follows to preview your version of the documentation on Read the Docs
...
+3. Go to `Read the Docs <https://readthedocs.org/>`_. Sign up for an account
+   if you don't have one.
 
 4. `Import <https://readthedocs.org/dashboard/import/>`_ the project.
 
 5. Enter the details of your project in the ``repo`` field and hit ``Create``.
 
-6. `Build your documentation <https://docs.readthedocs.io/en/stable/intro/import-guide.html#building-your-documentation>`_.
+6. `Build <https://docs.readthedocs.io/en/stable/intro/
+   import-guide.html#building-your-documentation>`_ your documentation.
```
Comment (terhorstd) [inline doc/documentation_workflow/user_documentation_workflow.rst:149]: does this work, or does this need to be single line?
```suggestion
6. `Build <https://docs.readthedocs.io/en/stable/intro/import-guide.html#building-your-documentation>`_ your documentation.
```

File: doc/documentation_workflow/user_documentation_workflow.rst [hunk 2]
```diff
@@ -114,17 +138,24 @@ Proceed as follows to preview your version of the documentation on Read the Docs
...
+to see the changes on the official NEST simulator documentation, please submit
+a pull request.
 
 Creating pull request
 +++++++++++++++++++++
 
-When you feel your documentation work is finished, you can create a `pull request <https://nest.github.io/nest-simulator/development_workflow#create-a-pull-request>`_ to the ``master`` branch of the NEST Source Code Repository. Your pull request will be reviewed by our NEST Documentation Team!
+Once your documentation work is finished, you can create a `pull request
+<https://nest.github.io/nest-simulator/development_workflow#create-a-pull-
+request>`_ to the ``master`` branch of the NEST Source Code Repository. Your
```
Comment (terhorstd) [inline doc/documentation_workflow/user_documentation_workflow.rst:160]: same here...

File: doc/documentation_workflow/user_documentation_workflow.rst [hunk 3]
```diff
@@ -87,13 +105,18 @@ Using Sphinx, you can build documentation locally and preview it offline:
 Editing and creating pages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To edit existing `reStructuredText <https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html>`_ files or to create new ones, follow the steps below:
+To edit existing `reStructuredText <https://thomas-cokelaer.info/tutorials/
+sphinx/rest_syntax.html>`_ files or to create new ones, follow the steps below:
```
Comment (terhorstd) [inline doc/documentation_workflow/user_documentation_workflow.rst:109]:
```suggestion
To edit existing `reStructuredText <https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html>`_
files or to create new ones, follow the steps below:
```

**Review by terhorstd — CHANGES_REQUESTED** [review]
Could you verify that the line breaks in the links don't cause them to break. My suggestions would bring them onto a single line, if necessary. Please mark as resolved otherwise.

**Comment by sarakonradi** [comment]
@terhorstd The links are unaffected! Just tested it again, and they all work.

**Review by terhorstd — APPROVED** [review]
OK, that's perfect! One less thing to worry about.

Thanks a lot!
:+1:

**Pull request merged**